apps/web-giddh/src/app/contact/contact.component.ts

```diff
@@ -315,22 +314,21 @@ export class ContactComponent implements OnInit, OnDestroy, OnChanges {
     public openBulkPaymentModal(template: TemplateRef<any>, item?: any): void {
         this.isBulkPaymentShow = true;
         this.selectedAccForPayment = null;
-        if (item && this.selectedAccountsList.length < 2) {
+        if (this.selectedAccountsList.length) {
+            this.selectedAccountsList = this.selectedAccountsList.filter(itemObject => {
```

Comment:
What is the use of this filter operation if the below filter operation will override it